mm2src/lp_swap/taker_swap.rs

```diff
@@ -1542,22 +1550,39 @@ pub async fn taker_swap_trade_preimage(
         Err(err) => return ERR!("!lp_coinfind({}): {}", other_coin_ticker, err),
     };
 
+    if req.max {
+        return ERR!("'max' cannot be used with 'sell' or 'buy' method");
```

Comment:
Could you please provide more details on why we require to "turn off" the max param for buy/sell?

Reply:
As we discussed with GUI developers, `trade_preimage` is called on the final order confirmation page when the `price` and `volume` are known already.
First, GUI should request `max_taker_vol` to filter the Maker orders.
Secondly, when the user has chosen a Maker order, GUI should request the trade preimage to inform the user of the expected fees.

Reply:
Ok, makes sense, thanks!